bin/ch/WScriptJsrt.cpp

```diff
@@ -275,17 +276,21 @@ JsValueRef WScriptJsrt::LoadScriptHelper(JsValueRef callee, bool isConstructCall
             }
         }
 
-        if (!fileNameNarrow && isSourceModule)
+        if(!fileNameNarrow)
```

Comment:
Style nitpick: we typically have a space between `if` and `(`.

Reply:
Done - sorry I should have noticed this from reading your code.